src/commands/commit/prompts.js

```diff
@@ -3,9 +3,8 @@ import inquirer from 'inquirer'
 
 import configurationVault from '../../utils/configurationVault'
 import filterGitmojis from '../../utils/filterGitmojis'
-import getDefaultCommitContent from '../../utils/getDefaultCommitContent'
-import { type CommitMode } from './index'
```

Comment:
Why we moved this outside of this module?

Reply:
Because I need the result of the function call for `getDefaultAnswers` and `prompts`. To avoid call it twice I moved it down to `commit/index`